bugrap/src/main/java/org/vaadin/bugrap/ui/ReportsView.java

```diff
@@ -155,4 +156,15 @@ private void revertChanges() {
 		model.resetReportForEdit();
 		binder.setBean(model.getReportForEdit());
 	}
+	
+	private void openReportDetail() {
+		Window window = new Window();
```

Comment:
Check Vaadin 8 docs, the Handling Browser Windows section. There's a component there very useful for this feature.

Reply:
I committed the changes to a new branch, but I think for this app, using normal Window is a better method

Reply:
Ok, but it would be better to have commited the changes in this branch.